packages/@aws-cdk/aws-eks/README.md

```diff
@@ -71,18 +79,12 @@ To disable the default capacity, simply set `defaultCapacity` to `0`:
 new eks.Cluster(this, 'cluster-with-no-capacity', { defaultCapacity: 0 });
 ```
 
-The `cluster.defaultCapacity` property will reference the `AutoScalingGroup`
+The `cluster.defaultCapacity` property will reference the `NodeGroup`
```

Comment:
Now We may need to have “defaultNodegroup” and “defaultCapacity” (for EC2) so people can access either of them.